lib/jest-helpers.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const nunjucks = require('nunjucks')
+const cheerio = require('cheerio')
+
+const COMPONENT_DIR = 'src/components'
+
+nunjucks.configure(COMPONENT_DIR, {
+  trimBlocks: true,
+  lstripBlocks: true
+})
+
+function render (name, params) {
```

Comment:
Would be great to add some comments in this file. For example, from the point of view of someone new to JS testing who's writing their first test for something they're contributing, I think it'd be useful to understand what Cheerio is and what it is doing, and how you use the return value when it returns two things `{ output, $ }`.

I'd also suggest maybe using `componentName` and `arguments` as the parameters. I _think_ we're starting to standardise on arguments for macros, but we should maybe document our terminology somewhere.

Reply:
Sounds good, do you think contributors will look at this file?

Isn't params what we're using everywhere? https://github.com/alphagov/govuk-frontend/search?utf8=%E2%9C%93&q=params&type=

Reply:
…yep.

Good point, well made.